doc/lua_api.txt

```diff
@@ -3219,6 +3220,7 @@ These functions return the leftover itemstack.
     * Returns true, if player `name` shouldn't be abled to dig at `pos` or do other
       actions, defineable by mods, due to some mod-defined ownership-like concept.
       Returns false or nil, if the player is allowed to do such actions.
+    * The name might not be associated with any particular player.
```

Comment:
Needs a reference to the empty string when called by mods.

Reply:
I can't think of any good reason to rely on knowing the player name used, but ok